modules/lang-painless/src/main/java/org/elasticsearch/painless/symbol/SemanticScope.java

```diff
@@ -38,11 +39,11 @@
  * its parent. As a scope is no longer necessary, it's dropped automatically
  * since parent scopes contain no references to child scopes.
  */
-public abstract class Scope {
+public abstract class SemanticScope {
```

Comment:
Do we wanna change any jdoc?

Reply:
I re-read through the JavaDocs in this file, and I think adding semantic in front of scope in most cases just makes it wordier without adding more information. The semantic phase is described as what the scopes in this class are for already.